webrtc/RTCCertificate.html

```diff
@@ -172,8 +172,9 @@
         assert_equals(typeof fingerprint, 'object',
           'Expect fingerprint to be an object (dictionary)');
 
-        // Can only do simple test as the allowed values may be extended
-        assert_true(/^[a-zA-Z\-]+$/.test(fingerprint.algorithm),
+        // https://www.iana.org/assignments/hash-function-text-names/hash-function-text-names.xml
+        const algorithms = ['md2', 'md5', 'sha-1', 'sha-224', 'sha-256', 'sha-384', 'sha-512'];
+        assert_in_array(fingerprint.algorithm, algorithms,
```

Comment:
The spec says "The browser selects the algorithm used to sign the certificate; a browser SHOULD select SHA-256 [FIPS-180-4] if a hash algorithm is needed."

Unless there is a known browser that does not use sha-256 maybe consider asserting that it is "sha-256", since this is `SHOULD`. Or do you think we should accept any of the known ones?

Also the spec's example includes `hash: "SHA-256"` with capital letters. Can you look up if it is defined if it should be upper or lower case or if both are acceptable?

Reply:
@henbos, do you think any changes to this PR are needed for this? Can you comment on the spec side?